script/upgrades/MU01/governance/MU01-CGP-Phase2.sol

```diff
@@ -238,6 +256,153 @@ contract MU01_CGP_Phase2 is ICGPBuilder, GovernanceScript {
     }
   }
 
+  function proposal_configureBreakerBox() public {
```

Comment:
We need to add all rate feed ids, including the new ones for EUR & BRL, to the mapping of monitored rate feed ids in the breaker box.
This was previously done in the initializer that was removed. This is done by calling the function `addRateFeeds`

Reply:
👍🏼